### PR TITLE
ShowWatchedList functionalities added

### PR DESCRIPTION
**Pull request by adzikrafi**
I added showWatchedList functionalities

### CODE REVIEW

File: src/main/java/MovieMate.java [hunk 1]
```diff
@@ -24,7 +23,7 @@ public static void main(String[] args) {
             switch (commandType) {
             case "watched":
                 // add to watched list
-                String[] movieInfo = ReadCSVFile.find(filePath, commandArg);
+                String[] movieInfo = ReadCSVFile.find("data/movies.csv", commandArg);
```
Comment (choongzhanhong) [inline src/main/java/MovieMate.java:26]: I'm thinking we can remove the first argument (filepath) and keep it as a constant in ReadCSVFile instead. To be refactored another time.

File: src/main/java/Ui.java [hunk 2]
```diff
@@ -21,4 +21,17 @@ public static void help() {
         System.out.println("tbc");
         System.out.println("Hope it helps, and have a nice day!");
     }
+
+    public static void showWatchedList(WatchedList watchedList){
+        System.out.println("Here is the list of movies that you have watched!");
```
Comment (choongzhanhong) [inline src/main/java/Ui.java:26]: Can move this line to under line 30?

File: src/main/java/Ui.java [hunk 3]
```diff
@@ -21,4 +21,17 @@ public static void help() {
...
+        if (watchedList.movieList.size()==0){
```
Comment (choongzhanhong) [inline src/main/java/Ui.java:27]: Can you put spaces between operators? e.g. `...size() == 0) {` for adherence to code style. Unsure why it wasn't flagged by checkstyle.

File: src/main/java/Ui.java [hunk 4]
```diff
@@ -21,4 +21,17 @@ public static void help() {
...
+            System.out.println("OOPS Sorry, It looks like you currently have no movie in the watchedList.");
+        }
+        else{
+            for(int i = 1; i <= watchedList.movieList.size();i++ ){
```
Comment (choongzhanhong) [inline src/main/java/Ui.java:31]: Same style comment here. Please help to fix any other discrepancies too. Thank you!

**Review by choongzhanhong — CHANGES_REQUESTED** [review]
Slight style changes.

Extracted method can be done in another PR. Thank you!

**Comment by adzikrafi** [comment]
Comment Addressed

**Pull request closed without merging**